### PR TITLE
Updated/Added missing areas to Targossas + Memory of Ember Tower Foray

### PR DESCRIPTION
**Pull request by ItsAeowynn**
## Checklist

The list below serves as a reminder to make sure, all needed changes were added to the repository. You can add a `x` between the square brackets to tick the items when in edit mode or just tick the checkboxes when not.

If you forgot something, don't worry. Just do the changes on the same branches and they are added automatically to the pull request.

- [X] Added changed map file
- [X] Used a descriptive pull request title
- [X] Wrote a short description of the change in the pull request description (see below). Include area names where possible.
- [X] Added a short sentence summarizing the changes to the `changelog.txt` file
- [X] Increased the number in `version.txt` by 1

## Included changes
 Targossas changes:
- Added Nacred Arcade and the Celestial Suites
- Updated shops + the museum layout

Added A memory of Ember Tower.

### CODE REVIEW

**Review by keneanung — COMMENTED** [review]
Thanks for your contribution! :tada: The changes look great! :star:

Before we merge, could you have a look at the room marks that are mentioned by the `Danger` bot above?

Thanks a lot!

**Review by keneanung — COMMENTED** [review]
Looks like you forgot to unlock Mhaldor :grimacing:

**Review by keneanung — APPROVED** [review]
❤️ thank you! I'll leave it sit 2 more days to let others have a look.

File: Map/changelog.txt [hunk 1]
```diff
@@ -455,4 +455,6 @@ changelog = {
 	[[New Pash-Jaru connection, Updated City shops]],
 	[[Added new rooms in Eleusis]],
 	[[Updated bog of Ashtan]],
+	[[Updated + added missing rooms in Targossas]]
```
Comment (vadi2) [inline Map/changelog.txt]: This is sneaky but need a comma here after ]], like in other lines

File: Map/version.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-458
+459
```
Comment (vadi2) [inline Map/version.txt]: I think this would need to be 460 because there are two new items in the changelog

**Review by vadi2 — APPROVED** [review]
Thx!!

**Pull request merged**